### PR TITLE
[DOCKER] Added a devel.Dockerfile for Docker users to deploy dev envs.

### PR DESCRIPTION
**Pull request by pkucmus**

### CODE REVIEW

File: requirements/devel.txt [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 ipython==4.1.1
 readline==6.2.4.1
-pdbpp==0.8.3
+pdbpp==0.8.1
```
Comment (pkucmus) [inline requirements/devel.txt]: Had to 0.8.3 has issues on Py3

Comment (jakubjanuzik) [inline requirements/devel.txt]: maybe `ipdb` then?

File: .gitignore [hunk 2]
```diff
@@ -62,3 +62,5 @@ target/
 .ipynb_checkpoints
 
 .vagrant
+apof/apof/settings/local.py
+db.sqlite3
```
Comment (jakubjanuzik) [inline .gitignore]: sqlite? not postgres? ( ͡° ͜ʖ ͡°)

**Comment by jakubjanuzik** [comment]
a few questions, except fine

File: requirements/devel.txt (hunk 1 above)
Comment (8r2y5) [inline requirements/devel.txt]: `ipdb` ftw

File: .gitignore (hunk 2 above)
Comment (8r2y5) [inline .gitignore]: u can't gitignore postgres, or do you?

**Comment by 8r2y5** [comment]
:+1:

File: requirements/devel.txt (hunk 1 above)
Comment (pkucmus) [inline requirements/devel.txt]: You can add if you want, leaving pdbpp either way

File: .gitignore (hunk 2 above)
Comment (pkucmus) [inline .gitignore]: He was asking in general... and yeah thought that leaving sqlite as a default is acceptable. We will use something more on prod but for this stage (no Docker-Compose, simple Vagrant) I'd leave it as is. Up to each dev if he want's to change it in `settings.local`.

**Pull request merged**